src/components/input-components.js

```diff
@@ -0,0 +1,23 @@
+// standard class with all states found in the other inputs of this work
+
+export default class Input {
```

Comment:
We already have a class with only enable end disable methods, it don't fit to your class? Can i give you some tips to you fix it?

Reply:
fixing in the other pr